packages/admin/src/snackbar/UndoSnackbar.tsx

```diff
@@ -0,0 +1,42 @@
+import { Button, Slide } from "@material-ui/core";
+import { TransitionProps } from "@material-ui/core/transitions";
+import * as React from "react";
+import { FormattedMessage } from "react-intl";
+
+import { useSnackbarApi } from "./SnackbarProvider";
+
+export interface SnackbarState<Payload extends unknown> {
+    key: React.Key;
+    message: React.ReactNode;
+    onActionButtonClick?: (payload?: Payload) => void;
+    payload?: Payload;
+}
+
+export const useUndoSnackbar = () => {
```

Comment:
There is no  real 'undo'-function implemented in the hook so the name "useUndoSnackbar" does not feel right to me.  From a hook called `useUndoSnackbar` I would expect a hook that manages `things` which can be un-done.

Can we think of a name that describes what the hook actually does?

Reply:
(Sorry for the late reply)
The intent of this hook is to provide a convenience abstraction of the snackbar for creating undo snackbars. This is useful since this will be the main use case for the snackbar (in fact, at the moment, it's the only use case 😅).

It would be possible to extend the api, make it more generic and rename it to `useMinimalSnackbar`, `usePreconfiguredSnackbar` or something like that. But I don't think there is really a need for this since you can simply use `useSnackbarApi` and customize it however you want.

My suggestion would be to move the `useUndoSnackbar` to `@comet/cms-admin` (because there we have multiple undo snackbars, that should look the same and feel the same and we want to prevent code duplication) and only leave the `useSnackbarApi` and `SnackbarProvider` in `@comet/admin`.

Reply:
Sounds reasonable. I'll remove the `useUndoSnackbar` hook and create a `UndoSnackbar` in `@comet/blocks-admin` then.

Reply:
Trouble is that such a general component doesn't really fit into `@comet/blocks-admin` - it's not related to blocks at all.

But for UndoSnackbar I don't see a reason why it should not live in `@comet/admin` - any non-cms application should use this pattern (if it makes sense)

Reply:
Yes true, after this discussion, it should live in  '@comet/admin'.

>I don't see a reason why it should not live

I saw no reason for this minimal component to live at all, as the component DOES nothing to deserve the name "Undo". But on second thought, since the component is a Snackbar and it renders a string "Undo", it should get it's place in the repo with name `UndoSnackbar`!